MC/bin/o2_hybrid_gen.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
```

Comment:
please add a short documentation what this tools does or achieves (including a short example usage)